src/main/java/ua/okwine/productexpirationdate/entity/Provider.java

```diff
@@ -0,0 +1,116 @@
+package ua.okwine.productexpirationdate.entity;
+
+
+
```

Comment:
one empty line will be enough. It can be fixed with auto-format settings if you use auto-format